### ISSUE TITLE
Pyopenms: Use full test suite in wheels actions and update building instructions

### ISSUE THREAD

**Issue by jpfeuffer** [OpenMS/OpenMS#6857]
see also #6855

**Comment by matteopilz** [comment]
Can we just use `ctest -R pyopenms` in wheels after we build?

**Comment by jpfeuffer** [comment]
Yes I think that should work.

**Comment by matteopilz** [comment]
Do we have/need a configuration for this?
I get failures in Linux, some exceptions in MacOS and in Windows it doesn't run, because of a missing -C.
https://github.com/matteopilz/OpenMS/actions/runs/6231850801/job/16914024553

**Comment by jpfeuffer** [comment]
Yes multiconfig generators like visual studio always need a `ctest -C Release ...` or Debug to specify which config to use while running tests.

**Comment by jpfeuffer** [comment]
For MacOS you probably should deactivate memory leak tests.
For Linux, I don't know, all tests failing is unlikely. It probably cannot load the shared libraries. But hard to say without the test output. Verbose Test output should probably be uploaded as artifact.

**Comment by timosachsenberg** [comment]
@matteopilz do you know what the state of this is?
